include/wx/notifmsg.h

```diff
@@ -81,6 +81,15 @@ class WXDLLIMPEXP_CORE wxNotificationMessageBase : public wxEvtHandler
         Timeout_Never = 0   // notification will never time out
     };
 
+    // reasons for dismissal, posted with wxEVT_NOTIFICATION_MESSAGE_DISMISSED events
+    enum class DismissalReason
```

Comment:
If we want to backport this to 3.2, we'd need to make this plain (unscoped) enum, e.g.

```cpp
enum DismissalReason
{
  Reason_Unknown,
  ...
};
```